public/app/features/dashboard/components/PanelEditor/utils.ts

```diff
@@ -41,26 +41,44 @@ export const updateDefaultFieldConfigValue = (
   isCustom?: boolean
 ) => {
   let defaults = { ...config.defaults };
-  const remove = value === undefined || value === null || '';
+  const remove = value == null || value === '';
```

Comment:
@dprokop `value === null || ''` looked like a mistake?

Reply:
@leeoniya no, it's not. We assume an empty string keeps no information and we just remove it from options.

Reply:
i mean the syntax, it looked wrong:

```js
const remove = value === undefined || value === null || '';
```

should have been maybe:

```js
const remove = value === undefined || value === null || value === '';
```

Reply:
🤦 of course it should :)

Reply:
never mind, i guess `if ('')` is same as `if (false)`. still i think having a boolean is better and more uniform.